plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSplitManager.java

```diff
@@ -77,7 +86,7 @@ public ConnectorSplitSource getSplits(
         }
 
         Table icebergTable = transactionManager.get(transaction, session.getIdentity()).getIcebergTable(session, table.getSchemaTableName());
-        Duration dynamicFilteringWaitTimeout = getDynamicFilteringWaitTimeout(session);
+        Duration dynamicFilteringWaitTimeout = config.getDynamicFilteringWaitTimeout();
```

Comment:
why?

Reply:
This was left here as a mistake, restored the original state. Thanks!